kube-linter.yaml

```diff
@@ -14,9 +14,9 @@ checks:
     - run-as-non-root
     - ssh-port
     - writable-host-mount
+    - read-only-root-fs
```

Comment:
do you mind moving this between privileged-container and run-as-non-root? this list is alphabetically sorted